modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/steps/StepsControlButtons.scala

```diff
@@ -94,9 +94,9 @@ object StepsControlButtons {
           case PauseObservation            =>
             Button(Button.Props(icon = Some(IconPause), color = Some("teal"), dataTooltip = Some("Pause the current exposure"), onClick = $.runState(handleObsPause(p.id, p.step.id))))
           case StopObservation             =>
-            Button(Button.Props(icon = Some(IconStop), color = Some("orange"), dataTooltip = Some("Stop the current exposure early"), disabled = s.stopRequested || SequenceState.internalStopRequested(p.sequenceState), onClick = $.runState(handleStop(p.id, p.step.id))))
+            Button(Button.Props(icon = Some(IconStop), color = Some("orange"), dataTooltip = Some("Stop the current exposure early"), onClick = $.runState(handleStop(p.id, p.step.id))))
```

Comment:
Enable/disable logic must be reworked. I removed it for now.

Reply:
In what sense?

Reply:
Like, when buttons should be enabled or disabled. I had to remove the disable for stop and abort, because the were disabled while the exposure was paused.